### PR TITLE
Add e2e test for timers

### PR DESCRIPTION
**Pull request by yuriadams**

### CODE REVIEW

File: examples/httpd/convox.yml [hunk 1]
```diff
@@ -2,3 +2,11 @@ services:
   web:
     image: httpd
     port: 80
+  worker:
+    build: ./worker
+    scale: 0
+timers:
+  cleanup:
```
Comment (Twsouza) [inline examples/httpd/convox.yml:9]: can you rename the timer to `example` or something more descriptive?
`cleanup` can be misleading.

File: examples/httpd/convox.yml [hunk 2]
```diff
@@ -2,3 +2,11 @@ services:
...
+    command: ./cmd.sh
+    schedule: "*/1 * * * ?"
+    service: worker
```
Comment (Twsouza) [inline examples/httpd/convox.yml:12]: Can you link the timer to the `web` service? So you won't need to create another service, keeping it simpler.

**Review by Twsouza — CHANGES_REQUESTED** [review]
I suggest you rename the timer and link it to the `web` service (and remove the worker).

**Review by heronrs — APPROVED** [review]
Looks good to me. Agree with @Twsouza on the comments.

File: examples/httpd/convox.yml (hunk 2 above)
Comment (yuriadams) [reply examples/httpd/convox.yml]: hey, @Twsouza, thank for your comments. Could you have a look again, please?

**Review by Twsouza — APPROVED** [review]
🔥

**Pull request merged**